graphene_django/management/commands/graphql_schema.py

```diff
@@ -60,8 +60,10 @@ def save_graphql_file(self, out, schema):
 
     def get_schema(self, schema, out, indent):
         schema_dict = {"data": schema.introspect()}
-        if out == "-":
+        if out == "-" or out == "-.json":
```

Comment:
Not sure if this is intuitive. Personally prefer an explicit option but I see that you are continuing an existing pattern, so.. :+1:

Reply:
I would prefer a command line flag too, but I decided to go with this since this was how the code was laid out first. I don`t wanna introduce any breaking changes since I just need this to work for my project :-)